src/interfaces/IOracleWriter.sol

```diff
@@ -2,14 +2,19 @@ pragma solidity 0.8.13;
 
 interface IOracleWriter {
     struct Observation {
-        // natural log (ln) of the price (token1/token0)
-        int112 logAccPrice;
-        // natural log (ln) of the liquidity (sqrt(k))
-        int112 logAccLiquidity;
+        // natural log (ln) of the raw accumulated price (token1/token0)
+        int112 logAccRawPrice;
+        // natural log (ln) of the clamped accumulated price (token1/token0)
+        // even in the case of extreme prices (3e75), will overflow once every 878 years
+        int56 logAccClampedPrice;
+        // natural log (ln) of the accumulated liquidity (sqrt(k))
+        // even in the case of extreme liquidity (3e75), will overflow once every 878 years
+        int56 logAccLiquidity;
         // overflows every 136 years, in the year 2106
         uint32 timestamp;
     }
 
-    function observations(uint256 index) external view returns (int112, int112, uint32);
+    function observations(uint256 index) external view returns (int112, int56, int56, uint32);
     function index() external returns (uint16);
+    function setMaxChangePerSecond(uint8) external;
```

Comment:
Is the reason for uint8 purely because the MAX < uint8.max? I would imagine this increases gas cost due to masking, given we store this as a uint64?

Reply:
this was an error on my part. Have changed the arg type to `uint64`, as that's the min to store `1e18`. 

Even if we use `uint64`, there still the masking problem right? Should I just use `uint256` instead?

Reply:
If we're not packing, then I would use uint256

Reply:
okay fixed.